Formula/k/kubetail.rb

```diff
@@ -0,0 +1,38 @@
+# typed: false
+# frozen_string_literal: true
+
+# This file was generated by GoReleaser. DO NOT EDIT.
+class Kubetail < Formula
+  desc "CLI tool with a web dashboard for viewing Kubernetes logs"
+  homepage "https://www.kubetail.com/"
+  url "https://github.com/kubetail-org/kubetail/archive/refs/tags/0.8.0-rc2.tar.gz"
+  version "0.0.1"
+  sha256 "6b6ac04a74a1724303e1c023ebd311d80ee10d13af922b4879a17bde4dc614ff"
+  license "Apache-2.0"
+
+  # Build dependencies
+  depends_on "go" => :build
+  depends_on "make" => :build
+  depends_on "node" => :build
+  depends_on "pnpm" => :build
+
+  def install
+    # Run the makefile build process if it orchestrates pnpm and go builds
+    system "make", "build"
+
+    # Install binary from `bin` folder
+    bin.install "bin/kubetail"
+  end
+
+  def post_install
+    generate_completions_from_executable(bin/"kubetail", "completion")
+  end
+
+  test do
+    # Verify that the kubetail binary is executable and returns expected output for --version
```

Comment:
```suggestion
```

Reply:
@chenrui333 Thanks for the feedback! I'll work on this tomorrow.

Reply:
Fixed